### DISCUSSION TITLE
Reorder Side Panel Buttons

### DISCUSSION THREAD

**Discussion by Geldahar** [runelite/runelite#16679] [Ideas]
The ability to reorder the side panel buttons such as for, XP tracker, Loot Tracker, Notes, etc

Some plugins add side panel buttons and there is currently no way to reorder them, which can cause the order to be sub optimal or change based on which plugins you are using.

Something like, Alt+Right Click > Move Up/Move Down/Move to Top/Move to Bottom, or Draggable with Left Click

**Comment by YvesW** [comment]
Duplicate of #15020
